pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -464,7 +464,7 @@ func (p *criStatsProvider) addPodCPUMemoryStats(
 			ps.CPU = &statsapi.CPUStats{}
 		}
 
-		ps.CPU.Time = cs.StartTime
+		ps.CPU.Time = cs.CPU.Time
```

Comment:
@Zheaoli  does this require a doc bug fix ? Where do these stats show up  for users ?

Reply:
I'm not sure about the doc. I will check it again

Reply:
I don't think we document this that specifically.

Reply:
This is for creating the Summary API.